### ISSUE TITLE
Undesired output while running meilix iso

### ISSUE THREAD

**Issue by meets2tarun** [fossasia/meilix#71]
Although meilix master branch is based on lxde but when I implemented the change in build.sh script to remove all the packages of lxde and included the packages of lxqt. But after booting the iso, it shows no change at all, even the desktop environment is shown to be of gnome.

**Comment by meets2tarun** [comment]
This is the place where I'm working https://github.com/meets2tarun/meilix/commits/history
This is the git diff of the changes I made
[diff.txt](https://github.com/fossasia/meilix/files/1117885/diff.txt)

**Comment by mariobehling** [comment]
@meets2tarun What are your actions to solve this issue? Please involve your co-developer and work as a team!

**Comment by abishekvashok** [comment]
I can't see where you did an update inside the chroot env. Keep in mind that, the things done inside he chroot env will only be present in the distro. Installing packages with apt get outside the chroot env, will cause the packages to be installed on the build system that is now Travis ci.

**Comment by abishekvashok** [comment]
Referencing #64

**Comment by meets2tarun** [comment]
The work done by me to show the changes are:

I first made required changes in the chromium-browser chrome.json file and then rename the folder chromium-browser to chromium.
Then delete the meilix-default-settings folder under the meilix-default-settings/debian and delete the all the metapackages files with meilix-default-settings_* under meilix repository.
I edited the changelog file under meilix-default-settings to version 1.0
Added the Makefile by `git checkout HEAD^ Makefile` in meilix-default-settings.
Then I went to folder meilix-default-settings and use `debuild -uc -us ` to rebuild the pacakges and it created all the files with the required changes which I just deleted earlier.
Then committed and testing.
